[2023-11-04 20:58:51,288] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: example_bash_operator.run_after_loop manual__2023-11-04T20:58:44.380375+00:00 [queued]>
[2023-11-04 20:58:51,308] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: example_bash_operator.run_after_loop manual__2023-11-04T20:58:44.380375+00:00 [queued]>
[2023-11-04 20:58:51,308] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2023-11-04 20:58:51,309] {taskinstance.py:1377} INFO - Starting attempt 1 of 1
[2023-11-04 20:58:51,309] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2023-11-04 20:58:51,331] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): run_after_loop> on 2023-11-04 20:58:44.380375+00:00
[2023-11-04 20:58:51,336] {standard_task_runner.py:52} INFO - Started process 150 to run task
[2023-11-04 20:58:51,340] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'example_bash_operator', 'run_after_loop', 'manual__2023-11-04T20:58:44.380375+00:00', '--job-id', '13', '--raw', '--subdir', '/home/***/.local/lib/python3.7/site-packages/***/example_dags/example_bash_operator.py', '--cfg-path', '/tmp/tmprsvczplc', '--error-file', '/tmp/tmpbs70qgp5']
[2023-11-04 20:58:51,341] {standard_task_runner.py:80} INFO - Job 13: Subtask run_after_loop
[2023-11-04 20:58:51,430] {task_command.py:371} INFO - Running <TaskInstance: example_bash_operator.run_after_loop manual__2023-11-04T20:58:44.380375+00:00 [running]> on host afb7adbe542c
[2023-11-04 20:58:51,510] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=example_bash_operator
AIRFLOW_CTX_TASK_ID=run_after_loop
AIRFLOW_CTX_EXECUTION_DATE=2023-11-04T20:58:44.380375+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-11-04T20:58:44.380375+00:00
[2023-11-04 20:58:51,512] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2023-11-04 20:58:51,512] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo 1']
[2023-11-04 20:58:51,523] {subprocess.py:85} INFO - Output:
[2023-11-04 20:58:51,524] {subprocess.py:92} INFO - 1
[2023-11-04 20:58:51,525] {subprocess.py:96} INFO - Command exited with return code 0
[2023-11-04 20:58:51,555] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=example_bash_operator, task_id=run_after_loop, execution_date=20231104T205844, start_date=20231104T205851, end_date=20231104T205851
[2023-11-04 20:58:51,593] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-11-04 20:58:51,625] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
